src/pages/Landing/index.tsx

```diff
@@ -109,7 +124,7 @@ export default () => {
               type='number'
             />
           </Grid>
-          <Grid className={classes.inlineText}>
+          <Grid className={`${classes.inlineText} ${darkmode ? 'dark-mode' : ''}`}>
```

Comment:
for these 3 changes 'dark-mode' isn't a class, and i believe they aren't necessary